crates/uv-toolchain/src/python_versions.inc.mustache

```diff
@@ -0,0 +1,28 @@
+// DO NOT EDIT
+//
+// Generated with `{{generated_with}}`
+// From template at `{{generated_from}}`
```

Comment:
Maybe I should generate this header fully in Python so there's not a DO NOT EDIT line in the template itself